shared/test/test_channels.rb

```diff
@@ -23,19 +23,36 @@ def test_create_channel
   end
 
   def test_update_channel
+    start = DateTime.now - 1
     post '/v3/channels', {abc: 123}.to_json, 'CONTENT_TYPE' => 'application/json;charset=utf-8'
     channel_id = last_response.location.split('/').last
 
     get "/v3/channels/#{channel_id}"
     assert last_response.ok?
-    assert_equal 123, JSON.parse(last_response.body)['abc']
+    result = JSON.parse(last_response.body)
+    assert_equal 123, result['abc']
 
+    # Check timestamps.
+    created = result['createdAt']
+    assert_equal created, result['updatedAt']
+    assert (start..DateTime.now).cover? DateTime.parse(created)
+
+    sleep 1
```

Comment:
Best practice is to use a mock clock that you can control arbitrarily, rather than physically sleeping.   (This enables a broader range of test scenarios and makes tests run quickly.)

The permissiveness of Ruby means that you can do this in a pretty non-invasive fashion, see for example Timecop.  (https://github.com/travisjeffery/timecop), which "provides a unified method to mock Time.now, Date.today, and DateTime.now in a single call."

Not saying you need to incorporate this for this PR, but worth considering for something more time-intensive.